src/features/components/atoms/card/card-primary/index.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import Typography from '../../typography'
+import Flex from '../../flex-box'
+import Link from '../../link'
+import { item_container, item_block } from './styles.module.scss'
+import { CardType } from './type'
+import { Localize } from 'components/localization'
+import useBreakpoints from 'components/hooks/use-breakpoints'
+interface CardItemsProps {
+    item: CardType
+}
+
+const CardBasic = ({ item }: CardItemsProps) => {
+    const { is_mobile_or_tablet } = useBreakpoints()
+    return (
+        <Flex.Box className={item_container} direction="col" gap="4x" bgcolor="white" radius="8x">
+            <Flex.Item className={item_block}>
+                {item.icon && (
+                    <Flex.Box pb="4x" lg={{ pb: '8x' }}>
+                        <img src={item.icon} width={44} height={44} />
```

Comment:
@aswathy-deriv I saw img here

Reply:
This is @meenakshi-deriv card component..I will check with her.Thank you